routing.go

```diff
@@ -226,14 +273,12 @@ func (dht *IpfsDHT) GetValues(ctx context.Context, key string, nvals int) ([]rou
 
 	// run it!
 	_, err = query.Run(ctx, rtp)
```

Comment:
@mildred right, i was mostly commenting that the existing behaviour might be incorrect. We can always change it in a different PR though.